### DISCUSSION TITLE
GP12 Long Term TL bug?

### DISCUSSION THREAD

**Discussion by PMyers21** [gopro/labs#659] [Q&A]
Hello, when I installed labs and connect the GP to a Voltaic V50 or a V75 battery its giving me a pop message saying the power source isn't powerful enough. 

My only current solution is to leave a battery in the GB, while connected to the V50. Is this a bug or does the GP need to stay in?

**Comment by dnewman-gpsw** [comment]
It is likely the cable not sending current ratings. This is the case on all camera since HERO10, and there is an easy fix.  !MTUSB=1 Trust USB power.

I do recommend leaving the battery in, particularly on HERO11 & 12, as there have been enhancement to support battery + external power. Battery in makes the camera's clock more reliable, so any time based actions will work over weeks or months. 

For long term timelapses, have a fully charged battery in the camera, and a V50 external power source. Optionally add Voltaic solar panel to keep the V50 charged. In this setup, the internal battery will always be at 100% charge, V50 is will be you main power, recharged or swapped out as needed.

**Reply by PMyers21** [reply]
Thanks for the info. 

By chance would you know why it reverts to "Nightlapse" for the photo settings vs. the regular photo mode?

**Reply by dnewman-gpsw** [reply]
I don't believe is does.  Timelapse Photo exposes to a maximum 1/8 of a seconds, whereas Nightlight auto exposes to up to 10seconds.  Nightlight is more flexible and recommended for all time-lapses.
